src/components/node-list/node-list-tree.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import TreeView from '@material-ui/lab/TreeView';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+
+import {
+  toggleModularPipelineActive,
+  toggleModularPipelineFilter,
+} from '../../actions/modular-pipelines';
+import { toggleTypeDisabled } from '../../actions/node-type';
+import { getNodeTypes, getNodeTypeIDs } from '../../selectors/node-types';
+import {
+  getModularPipelineIDs,
+  getModularPipelineData,
+} from '../../selectors/modular-pipelines';
+import { getNestedModularPipelines } from './node-list-items';
+import {
+  getGroupedNodes,
+  getNodeSelected,
+  getNodeModularPipelines,
+} from '../../selectors/nodes';
+import { loadNodeData } from '../../actions/nodes';
+import NodeListTreeItem from './node-list-tree-item';
+
+const useStyles = makeStyles({
```

Comment:
Is there a way to apply these styles through the sass files? Just thinking about how to keep consistent with the other components if possible

Reply:
`makeStyles` is an interpolation provided by material-ui for quick access for quick style modifications to their pre-set components -I've opted to use it here for quick set up of the tree-list as it saves us the need to introduce and maintain any overwrites to their pre-set css classes. This will only be used as a one-off setup here and will provide minimal disruption to how we write and maintain our styling within the app.